src/query/storages/stage/src/append/parquet_file/writer_processor.rs

```diff
@@ -83,6 +84,7 @@ fn create_writer(
         .set_dictionary_enabled(false)
         .set_statistics_enabled(EnabledStatistics::None)
         .set_bloom_filter_enabled(false)
+        .set_created_by(DATABEND_COMMIT_VERSION.to_string())
```

Comment:
Why do we need using `DATABEND_COMMIT_VERSION` as parquet `created_by`?

Reply:
Supplemented in the summary